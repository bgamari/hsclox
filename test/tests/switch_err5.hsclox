switch (1) {
    default: print "hi";
