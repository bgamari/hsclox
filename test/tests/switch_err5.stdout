[line 3] Error at end: Expect closing '}'.
